xmsgrid/python/ugrid/XmUGridUtils_py.cpp

```diff
@@ -21,26 +21,76 @@ namespace py = pybind11;
 PYBIND11_DECLARE_HOLDER_TYPE(T, boost::shared_ptr<T>);
 
 void initXmUGridUtils(py::module &m) {
-    py::module modXmUGridUtils = m.def_submodule("XmUGridUtils")
-      .def("xm_read_ugrid_from_ascii_file", &xms::XmReadUGridFromAsciiFile,
-        "Read XmUGrid from an ASCII file.",py::arg("file_path"))
-      .def("xm_write_ugrid_to_ascii_file", &xms::XmWriteUGridToAsciiFile,
-        "Write an XmUGrid to an ASCII file.",py::arg("ugrid"),py::arg("file_path"))
-      .def("cross", [](py::iterable Origin, py::iterable A, py::iterable B) -> double {
+  py::module modXmUGridUtils = m.def_submodule("XmUGridUtils");
+  // ---------------------------------------------------------------------------
+  // function: xm_read_ugrid_from_ascii_file
+  // --------------------------------------------------------------------------- 
+  const char* xm_read_ugrid_from_ascii_file_doc = R"pydoc(
```

Comment:
Do we want to get rid of the xm off of this function? None of the other functions use that prefix